### ISSUE TITLE
Bit accuracy remains very low when training on 4×RTX 4090 (24GB) with the 50K SA-1B dataset

### ISSUE THREAD

**Issue by yuxuan02315** [facebookresearch/videoseal#80]
Hi, thanks for your great work and for open-sourcing the code.

I'm trying to reproduce your results, but I'm encountering an issue during training.

According to the paper, the experiments were conducted using 8×H200 GPUs. Since I only have access to 4×RTX 4090 (24GB) GPUs, I trained the model on the 50K SA-1B dataset instead. I kept all other settings the same as the default configuration.

However, during training, the bit accuracy stays very low and barely improves.

I would like to ask:

- Is the model particularly sensitive to the number or type of GPUs?
- Is there a recommended learning rate scaling rule or any other hyperparameters that should be adjusted when training on fewer GPUs?
- Has anyone successfully reproduced the reported bit accuracy using 4×4090 GPUs?

My training environment is:

- GPUs: 4 × RTX 4090 (24GB)
- Dataset: SA-1B (50K subset)
- Configuration: Same as the default settings unless required by GPU memory
- Distributed training: DDP

Any suggestions would be greatly appreciated. Thanks!

**Comment by pierrefdz** [comment]
Hi @yuxuan02315 

Can you give the commands you ran?

**Comment by yuxuan02315** [comment]
> Hi [@yuxuan02315](https://github.com/yuxuan02315)
> 
> Can you give the commands you ran?

My training command is almost the same as the official setting:
`torchrun --nproc_per_node=$NGPU train.py --local_rank 0 \
    --video_dataset none --image_dataset myimages --workers 8 \
    --extractor_model convnext_tiny --embedder_model unet_small2_yuv_quant --hidden_size_multiplier 1 --nbits 256 \
    --scaling_w_schedule Cosine,scaling_min=0.2,start_epoch=200,epochs=200 --scaling_w 1.0 --scaling_i 1.0 --attenuation jnd_1_1 \
    --epochs 601 --iter_per_epoch 1000 --scheduler CosineLRScheduler,lr_min=1e-6,t_initial=601,warmup_lr_init=1e-8,warmup_t=20 --optimizer AdamW,lr=5e-4 \
    --lambda_dec 1.0 --lambda_d 0.1 --lambda_i 0 --perceptual_loss yuv --num_augs 2 --augmentation_config configs/all_augs.yaml --disc_in_channels 1 --disc_start 100 \
    --batch_size 32 --batch_size_eval 32 \
    --output_dir output/videoseal_img_256b_official`

**Comment by pierrefdz** [comment]
Hmm ok, I think it looks ok.

> However, during training, the bit accuracy stays very low and barely improves.

The bit accuracy stays at 0.5 during the whole training? If yes, what you can try smaller LR maybe, cut the run if it never starts. Tell me if after doing this you still have the issue
